content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -388,8 +388,8 @@ to 1 second. Minimum value is 1.
 * `successThreshold`: Minimum consecutive successes for the probe to be
 considered successful after having failed. Defaults to 1. Must be 1 for liveness
 and startup Probes. Minimum value is 1.
-* `failureThreshold`: When a probe fails, Kubernetes will
-try `failureThreshold` times before giving up. Giving up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready.
+* `failureThreshold`: When a probe fails `failureThreshold` times in a row, Kubernetes will give up.
+Give up in case of liveness probe means restarting the container. In case of readiness probe the Pod will be marked Unready.
```

Comment:
I think the main issue precisely the phrasing. I could append more text to minimize confusion but it will never make it crystal clear.  
What's pushing you towards appending text rather than modifying it?